apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -355,6 +360,20 @@ public void log(Request request, Response response) {
             HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES,
             String.valueOf(response.getHttpOutput().getWritten()));
 
+        String responseDurationInMilliseconds =
+            (String) request.getAttribute("response_duration_milliseconds");
+
+        if (response.getHttpOutput().getWritten() != 0
+            && !Strings.isNullOrEmpty(responseDurationInMilliseconds)) {
+          long responseDurationPerKB =
+              ((1024 * Long.parseLong(responseDurationInMilliseconds))
+                  / response.getHttpOutput().getWritten());
+          BfdMDC.put(
+              MDC_HTTP_ACCESS_RESPONSE_DURATION_PER_KB, String.valueOf(responseDurationPerKB));
+        } else {
+          BfdMDC.put(MDC_HTTP_ACCESS_RESPONSE_DURATION_PER_KB, null);
+        }
+
         /*
          * Write to the access.json. The message here isn't actually the payload; the MDC context that will get
          * automatically included with it is!
```

Comment:
we should add a check to the IT for this class. its not great validation but we can at least check that this new field got populated as we do for the output size.

Reply:
done